lib/SILOptimizer/Transforms/ArrayElementValuePropagation.cpp

```diff
@@ -53,18 +57,25 @@ class ArrayAllocation {
 
   // The calls to Array get_element that use this array allocation.
   llvm::SmallSetVector<ApplyInst *, 16> GetElementCalls;
+  llvm::SmallVector<ApplyInst *, 4> AppendContentsOfCalls;
   llvm::DenseMap<uint64_t, SILValue> ElementValueMap;
+  llvm::SmallVector<SILValue, 4> ElementValueVector;
```

Comment:
Does this need to be a class member?
Can't you just create this vector before you use it in ArrayAllocation::findReplacements()?